core/src/main/java/org/mskcc/cbio/portal/util/ExtendedMutationUtil.java

```diff
@@ -45,7 +47,9 @@
  */
 public class ExtendedMutationUtil
 {
-	public static final String NOT_AVAILABLE = "NA";
+	// originally NA (case ignored) was the only value for empty values in this column, but [Not Available]
+	// occurs in many TCGA studies.
+	public static final List <String> NOT_AVAILABLE = Arrays.asList("NA", "[Not Available]");
```

Comment:
If we're going to have a bunch of these that could potentially grow, maybe we should use an enum as in the [clinical data loader](https://github.com/cBioPortal/cbioportal/blob/master/core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java#L66). Just a suggestion, or maybe can be a TODO for later. I don't think it needs to block merging.